test/bin/pyutils/common.py

```diff
@@ -163,7 +174,7 @@ def read_file(file_path: str):
 def append_file(file_path: str, content: str):
     """Append the specified content to a file"""
     with open(file_path, 'a') as file:
-        file.write(content)
+        file.write(content + '\n')
```

Comment:
Nit: This is a generic function and we might want to use it for other purpose. Let's not add `\n` unconditionally, but when it's called in record_junit?